tests/braille-specs/Makefile.am

```diff
@@ -123,6 +123,9 @@ EXTRA_DIST =					\
 	sw-ke.yaml				\
 	ta-ta-g1_harness.yaml			\
 	th.yaml					\
+	tr-8dots_harness.yaml					\
+		tr-g1_harness.yaml					\
+		tr-g2_harness.yaml					\
```

Comment:
I prefer that the names of the yaml files to be just `tr-8dots.yaml`, i.e. drop the `harness` part. I don't think it adds any value

Reply:
Great, excellent